search/tests/test_indexing.py

```diff
@@ -36,3 +38,59 @@ def test_strip_xml(self):
         self.resource.save()
         self.assertTrue(self.count_results("you're it") == 1)
         self.assertTrue(self.count_results(xml) == 0)
+
+    def test_sorting(self):
```

Comment:
Note: Comment above has been addressed, but it didn't require changing line 42, so it didn't get hidden.